app/views/osbbs/_form.html.slim

```diff
@@ -0,0 +1,27 @@
+- flash.each do |type, msg|
+      div
+        = msg
+= simple_form_for @osbb do |f|
```

Comment:
do some alignment in the form or at least put label on top of the input
looks 
otherwise it looks like a mess